src/doc/unstable-book/src/compiler-flags/profile.md

```diff
@@ -12,10 +12,15 @@ For example:
 ```Bash
 cargo new testgcov --bin
 cd testgcov
-export RUSTFLAGS="-Zprofile"
+export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort"
+export CARGO_INCREMENTAL=0
 cargo build
 cargo run
 ```
 
 Once you've built and run your program, files with the `gcno` (after build) and `gcda` (after execution) extensions will be created.
 You can parse them with [llvm-cov gcov](https://llvm.org/docs/CommandGuide/llvm-cov.html#llvm-cov-gcov) or [grcov](https://github.com/mozilla/grcov).
+
+Please note that `RUSTFLAGS` apply to everything that cargo builds and runs during a build, including build scripts!
+To avoid this, pass a `RUSTC_WRAPPER` program to cargo that only adds the profiling flags to rustc for the specific 
+crates you want to profile.
```

Comment:
When `--target` flag is passed to cargo, the `RUSTFLAGS` no longer apply to build scripts and procedural macros. This could be more convenient to use than a wrapper.

Reply:
Thanks for the feedback. I added a sentence regarding `--target` and mentioned `RUSTC_WRAPPER` as an option if one wants more control over which crates get instrumented.